app/controllers/kondos_controller.rb

```diff
@@ -3,11 +3,23 @@ class KondosController < ApplicationController
   skip_before_action :authenticate_user!, only: [:index, :show]
 
   def index
-    unless params[:search_kondos] == ""
-      @kondos = @kondos.where("LOWER(prefecture) like ?","%#{params[:search_kondos].to_s.downcase}%")
-    else
-      @kondos = Kondo.all
+  
+    #Renter's Page
+    if current_user.renter?
```

Comment:
Here too we may want to add a third conditional - if user_signed_in? is false then we can display all kondos. What do you think? (Please let me know if I am missing something lol, I am not very confident on this)